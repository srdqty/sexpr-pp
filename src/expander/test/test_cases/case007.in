(lambda (x)
  (let-syntax
  (
    (y  (lambda (stx)
          (car (cdr (stx-e stx)))))
    (z (lambda (stx) (syntax x)))
  )
    (lambda (x)
      ((y (z)) 1 2 3 4))))
