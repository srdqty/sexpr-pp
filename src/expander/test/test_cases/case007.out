>> (lambda (x0) (begin (lambda (x1) (x0 1 2 3 4))))
>> 
